flows/fields.go

```diff
@@ -102,7 +102,10 @@ func (v *FieldValue) Reduce(env utils.Environment) types.XPrimitive {
 }
 
 // ToXJSON is called when this type is passed to @(json(...))
-func (v *FieldValue) ToXJSON(env utils.Environment) types.XText { return v.Reduce(env).ToXJSON(env) }
+func (v *FieldValue) ToXJSON(env utils.Environment) types.XText {
```

Comment:
unrelated bug that I found because we can now test @JSON() calls on things with time values - the field value might return `nil` from `Reduce`